.gitignore

```diff
@@ -7,3 +7,4 @@ example/*.js
 example/*.map
 test/*.js
 test/*.map
+TEST_2G
```

Comment:
That's just a folder of for some parsers I was trying. It contains various CSS and HTML parsers I was looking at while making my proposal. I'll remove that in the next commit.